Tests/UnitTests/Core/Propagator/VolumeMaterialInteractionTests.cpp

```diff
@@ -28,12 +28,21 @@ using namespace Acts::UnitLiterals;
 namespace Acts {
 namespace Test {
 
+/// @brief Mockup ConstraintStep
+struct ConstrainedStep {
+  ActsScalar step = 0.;
+
+  ActsScalar value() const { return step; }
+};
+
```

Comment:
would it make sense to include the existing constrained step here as it is quite simplistic already

Reply:
That's just to make less dependency - no hard feelings though.